src/seedu/addressbook/AddressBook.java

```diff
@@ -228,6 +229,7 @@ public static void main(String[] args) {
 
     private static void showWelcomeMessage() {
         showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
+        //System.out.println();//modify the behaviour slightly
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.